manager/resolver/maven/src/main/java/org/jboss/forge/furnace/manager/maven/util/MavenRepositories.java

```diff
@@ -27,13 +28,16 @@
 public class MavenRepositories
 {
    private static final String MAVEN_CENTRAL_REPO = "http://repo1.maven.org/maven2";
+   private static final List<String> CENTRAL_MIRRORS_IDS= Arrays.asList("central","*");
 
    public static List<RemoteRepository> getRemoteRepositories(MavenContainer container, Settings settings)
    {
       Set<RemoteRepository> remoteRepos = new HashSet<>();
       remoteRepos.addAll(container.getEnabledRepositoriesFromProfile(settings));
-      // If no remote repositories found or no mirrors to central found, add central
-      if (remoteRepos.isEmpty())
+
+      // central repository is added if there is no central mirrors
+      long numberOfCentralMirrors = settings.getMirrors().stream().filter(m -> CENTRAL_MIRRORS_IDS.contains(m.getId())).count();
```

Comment:
Also, will `remoteRepos` be non-empty if a mirror exists? We can't return an empty list in this method